### PR TITLE
describe tests in README; use jest projects

### PR DESCRIPTION
**Pull request by gregdyke**
npm test now runs all tests. Running only the mandatory tests is described in the README.

Suggestion for when we have auto running of tests on commit would be to add an npm script that is run with `npm run post-commit` or something like that (that would only run the mandatory tests and not the extra?)

-----
[View rendered README.md](https://github.com/CodeYourFuture/JavaScript-Core-1-Coursework-Week1/blob/readme_js1wk1/README.md)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,6 +6,26 @@ The exercises are split into three folders: `exercises`, `mandatory` and `extra`
 
 The `extra` folder contains exercises that you can complete to challenge yourself, but are not required for the following lesson.
 
+## Running the code/tests
+
+These files for the mandatory/extra exercises are intended to be run as jest tests.
```
Comment (40thieves) [inline README.md:11]: I assume this is was is intended?

```suggestion
The files for the mandatory/extra exercises are intended to be run as jest tests. 
```

File: README.md [hunk 2]
```diff
@@ -6,6 +6,26 @@ The exercises are split into three folders: `exercises`, `mandatory` and `extra`
...
+
+For more information about tests, look here:
+
+https://syllabus.codeyourfuture.io/guides/intro-to-tests
+
+Try out variant way of running tests:
+
+- `npm test` -> run all mandatory and extra tests
+- `npm test -- --selectProjects mandatory` -> run only mandatory tests
+- `npm test -- --testPathPattern mandatory/1-syntax-errors.js` -> run single test
```
Comment (40thieves) [inline README.md:27]: This feels repetitive with the section above, what is it adding?

File: README.md [hunk 3]
```diff
@@ -6,6 +6,26 @@ The exercises are split into three folders: `exercises`, `mandatory` and `extra`
...
 
+## Running the code/tests
+
+These files for the mandatory/extra exercises are intended to be run as jest tests. 
+
+- Once you have cloned the repository, run `npm install` once in the terminal to install jest (and any necessary dependencies).
+- To run the tests for all mandatory/extra exercises, run `npm test`
+- To run only the tests for the mandatory exercises, run `npm test -- --selectProjects mandatory`
+- To run only the tests for the extra exercises, run `npm test -- --selectProjects extra`
+- To run a single exercise/test (for example `mandatory/1-writer.js`), run `npm test -- --testPathPattern mandatory/1-writer.js` (Remember, you can use tab-completion to get files relative to the current directory, so m`Tab ↹`/1-`Tab ↹` will autocomplete get you the test file starting with 1-)
```
Comment (40thieves) [inline README.md:17]: What do you think about using `<kbd>`? Perhaps a bit overly clever?

```suggestion
- To run a single exercise/test (for example `mandatory/1-writer.js`), run `npm test -- --testPathPattern mandatory/1-writer.js` (Remember, you can use tab-completion to get files relative to the current directory, so <kbd>m</kbd><kbd>Tab ↹</kbd>/<kbd>1-</kbd><kbd>Tab ↹</kbd> will autocomplete get you the test file starting with 1-)
```

That would render as: <kbd>m</kbd><kbd>Tab ↹</kbd>/<kbd>1-</kbd><kbd>Tab ↹</kbd>

**Review by 40thieves — APPROVED** [review]
Some suggestions, otherwise looking good 🙂

**Review by illicitonion — APPROVED** [review]
LGTM - thanks!

**Comment by 40thieves** [comment]
We have stalebot problems!

**Comment by ChrisOwen101** [comment]
@40thieves Seems like it's working exactly as intended to me 😉

**Pull request merged**